README.md

```diff
@@ -1,24 +1,201 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon ⌛
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby
+- Ruby on Rails
+- PostgreSql
+
+> Visit this [link](https://www.ruby-lang.org/en/documentation/installation/) on how to install Ruby for different OS's.
+
+> Visit this [link](https://www.postgresql.org/download/) on how to install PostgreSQL.
+
+After installing Ruby you can install rails by using the following command:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  git clone https://github.com/uisendy/blog-app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog-app
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+- No tests yet
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+
+--->
+
+### Deployment
+
+- N/A
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Sendy Inieke**
+
+- GitHub: [@AbelG101](https://github.com/uisendy)
+- LinkedIn: [sendyinieke](https://www.linkedin.com/in/sendyinieke/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Provide live demo]**
+- [ ] **[Multi blogging option]**
+- [ ] **[Viewers analytics based on location]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project make sure to give it a star 🌟
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Great job @uisendy 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.